public-src/workers.jsx

```diff
@@ -29,6 +31,16 @@ const Workers = React.createClass({
     clearTimeout(this.timer)
   },
 
+  handleAutoRemoveWorkers(){
```

Comment:
I don't think this should be `auto`, but you can have a button for it (with a confirm)